### PR TITLE
Remove runc package and replace with containerd.io

### PR DESCRIPTION
**Pull request by thanos1983**
Installing CRI-O by source on Centos7 gives the following warning:

```bash
Package runc-1.0.0-67.rc10.el7_8.x86_64 is obsoleted by containerd.io-1.2.13-3.1.el7.x86_64 which is already installed
```

And also fails with error:

```bash
-- Unit crio-wipe.service has begun starting up.
Jul 31 14:31:30 domain bash[32278]: /usr/libexec/crio/crio-wipe/lib.bash: line 26: "1" + 0: syntax error: operand expected (error token is ""1" + 0")
Jul 31 14:31:30 domain systemd[1]: crio-wipe.service: main process exited, code=exited, status=1/FAILURE
Jul 31 14:31:30 domain systemd[1]: Failed to start CRI-O Auto Update Script.
-- Subject: Unit crio-wipe.service has failed
```

The solution is to install the latest version of containerd.io package.

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind dependency-change
> /kind deprecation
> /kind design

documentation

> /kind failing-test
> /kind feature
> /kind flake

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by haircommander** [comment]
gah, I am not pumped about this change, as it implies we need containerd to run cri-o (which we do not).
also, I don't know if this works for rhel {7,8} or any version of fedora

**Comment by haircommander** [comment]
hm I just ran a centos:7 container on my host, and runc installed fine and I did not get the obsoleted message. What version of centos and what repos are you subscribed to?

Ultimately, this section is very out of date. for centos 7 we recommend users use our OBS repos instead.

**Comment by thanos1983** [comment]
Hello again haircommander,

My mistake I should update the pull to RH7 instead of Centos7.

I will give you the exact version of CRI-O on Monday and the version of RH7.

I am doing everything by source so if I remember correctly I am running one version before the latest (not 100%) sure though.

Let me know if you require any further information.

Best Regards,
Thanos

File: tutorials/setup.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ yum install -y \
   libselinux-devel \
   pkgconfig \
   make \
-  runc
+  containerd.io
```
Comment (saschagrunert) [inline tutorials/setup.md:63]: Out of curiosity: what is included in this package?

**Comment by giuseppe** [comment]
NACK, let's fix the packaging if required

**Comment by rhatdan** [comment]
Definitely not.  If this is an issue in the distro it should be fixed there.

**Pull request closed without merging**